serif_shared/src/commonMain/kotlin/xyz/room409/serif/serif_shared/MatrixClient.kt

```diff
@@ -83,11 +83,24 @@ class MatrixSession(val client: HttpClient, val access_token: String, var transa
             return Error("Message Send Failed", e)
         }
     }
+    fun sendReadReceipt(eventId: String, room_id: String): Outcome<String> {
+        try {
+            val result = runBlocking {
```

Comment:
The inner block doesn't return a value the way it is now, so the var result isn't doing anything & we might want to remove it